### PR TITLE
Add camping icons

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

**Review by ericfennis — APPROVED** [review]
Really lovely icons!

**Comment by jguddas** [comment]
Where I'm from this is a common form of camping shelter, probably hard to create an icon for tho.

https://wiki.openstreetmap.org/wiki/Tag:shelter_type%3Dlean_to

File: icons/tent-tree.svg
Comment (karsa-mistmere) [inline icons/tent-tree.svg:1]: Looks really nice in general, but the central line is off grid and is quite blurry at 100%:
image

File: icons/tipi.svg
Comment (karsa-mistmere) [inline icons/tipi.svg:1]: I think the original name is just fine – while the icon does look very tipi-ish, a stylized tent would not look that much different.

**Comment by JClackett** [comment]
Working on a camper van icon, let me know if this could be added, open to changes of course!

Screenshot 2023-07-23 at 10 41 06

File: icons/tent.svg [hunk 1]
```diff
@@ -9,9 +9,8 @@
...
   stroke-linejoin="round"
 >
-  <path d="M19 20 10 4" />
-  <path d="m5 20 9-16" />
+  <path d="M4.5 20 14 4" />
+  <path d="M19.5 20 10 4" />
   <path d="M3 20h18" />
-  <path d="m12 15-3 5" />
-  <path d="m12 15 3 5" />
+  <path d="m15 20-3-5-3 5" />
```
Comment (karsa-mistmere) [inline icons/tent.svg:15]: I think we could easily bump this up in size to better match our ideal visual weight:
image
```svg
  <path d="M3.5 21 14 3" />
  <path d="M20.5 21 10 3" />
  <path d="M2 21h20" />
  <path d="M15.5 21 12 15l-3.5 6" />
```

What do you think?

File: icons/flame-kindling.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M15.5 7c-1-1-2.5-2-3.5-5-1 0-4 3-1.6 5.6a2 2 0 1 1-3.3 2c-.1.3-.1.6-.1.9A4.95 4.95 0 1 0 15.5 7" />
```
Comment (karsa-mistmere) [inline icons/flame-kindling.svg:12]: Looks pretty neat at 2px, but the bottom part is off grid and appears quite blurry at 1x.
```suggestion
  <path d="M15.5 6.5C14.5 5.5 13 5 12 2c-1 0-4 2.5-1.6 5.1a2 2 0 1 1-3.3 2c-.1.3-.1.6-.1.9a5 5 0 1 0 8.5-3.5" />
```
image

Comment (danielbayley) [reply icons/flame-kindling.svg:12]: @karsa-mistmere Ah yeah, it got over-optimised by https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit

**Comment by danielbayley** [comment]
> Working on a camper van icon, let me know if this could be added, open to changes of course!

@JClackett Nice, feel free to open another PR… See [`bus`](https://lucide.dev/icons/bus) and https://github.com/lucide-icons/lucide/pull/1412#issuecomment-1612434374 for help with curvature.

File: icons/flame-kindling.svg (hunk 2 above)
Comment (karsa-mistmere) [reply icons/flame-kindling.svg]: Not sure if this should be fixed by 42fbffb, but it still looks the same for me:
image

**Review by jguddas — APPROVED** [review]

File: icons/flame-kindling.svg [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M15.5 7c-1-1-2.5-2-3.5-5-1 0-4 3-1.6 5.6a2 2 0 1 1-3.3 2c-.1.3-.1.6-.1.9A5 5 0 1 0 15.5 7" />
+  <path d="m5 22 14-4" />
+  <path d="m5 18 14 4" />
```
Comment (karsa-mistmere) [inline icons/flame-kindling.svg:14]: I'm still not happy with how blurry the bottom of the flame is, but nonetheless, this one's still over-optimised:

```suggestion
  <path d="M12 2c1 2.98 2.53 3.97 3.5 5a5 5 0 1 1-8.42 2.58 2 2 0 1 0 3.3-2.02A3.55 3.55 0 0 1 12 2" />
  <path d="m5 22 14-4" />
  <path d="m5 18 14 4" />
```

Comment (ericfennis) [reply icons/flame-kindling.svg:14]: This one has a blurry line as well. I've made an optimized one

**Review by karsa-mistmere — APPROVED** [review]

**Pull request merged**